drafttopic/utilities/fetch_wikiprojects.py

```diff
@@ -112,114 +111,95 @@ def parseWpDirectory(self):
                                 'root_url': sec['fromtitle'],
                                 'index': sec['index']}
             # Get entries in this section
-            self.logger.info("Fetching entries for section:{}".format(name))
-            section = self.getSectionText(self.session, dirname, sec['index'])
+            self.logger.info("Fetching entries for section :{}".format(name))
+            section = self.getSectionText(dirname, sec['index'])
             mainHeading = re.search(WPMainHeadingRegex, section)
             if mainHeading:
                 wp[sec['line']]['url'] = WPDPage + '/' + mainHeading.group(1)
-                wp[sec['line']]['topics'] = self.getSubCategories(self.session,
-                                                            wp[sec['line']]['url'])
-
-
+                subPageSections = self.getSections(wp[sec['line']]['url'])
```

Comment:
Variable names should be snake case (lower_case_with_underscores) https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles